packages/core/src/routes/application.ts

```diff
@@ -50,7 +50,7 @@ export default function applicationRoutes<T extends AuthedRouter>(router: T) {
         type,
         name,
         oidcClientMetadata: buildOidcClientMetadata(oidcClientMetadata),
-        customClientMetadata,
+        customClientMetadata: customClientMetadata ?? {},
```

Comment:
Why do we need to fallback undefined to `{}` on server? This should've be done on client side

Reply:
this code is for local debug only, added by accident, need to remove.

Reply:
ok